src/quo2/components/animated_header_flatlist/view.cljs

```diff
@@ -1,97 +0,0 @@
-(ns quo2.components.animated-header-flatlist.view
```

Comment:
Please put back `animated_header_list` files, they will be used for communities, profile, and other un-inverted lists